test/fail_compilation/reserved_version.d

```diff
@@ -2,101 +2,103 @@
+fail_compilation/reserved_version.d(178): Error: version identifier `Alpha_SoftFloat` is reserved and cannot be set
+fail_compilation/reserved_version.d(179): Error: version identifier `Alpha_HardFloat` is reserved and cannot be set
+fail_compilation/reserved_version.d(180): Error: version identifier `LittleEndian` is reserved and cannot be set
+fail_compilation/reserved_version.d(181): Error: version identifier `BigEndian` is reserved and cannot be set
+fail_compilation/reserved_version.d(182): Error: version identifier `ELFv1` is reserved and cannot be set
+fail_compilation/reserved_version.d(183): Error: version identifier `ELFv2` is reserved and cannot be set
+fail_compilation/reserved_version.d(184): Error: version identifier `CRuntime_Bionic` is reserved and cannot be set
+fail_compilation/reserved_version.d(185): Error: version identifier `CRuntime_DigitalMars` is reserved and cannot be set
+fail_compilation/reserved_version.d(186): Error: version identifier `CRuntime_Glibc` is reserved and cannot be set
+fail_compilation/reserved_version.d(187): Error: version identifier `CRuntime_Microsoft` is reserved and cannot be set
+fail_compilation/reserved_version.d(188): Error: version identifier `CRuntime_Musl` is reserved and cannot be set
+fail_compilation/reserved_version.d(189): Error: version identifier `CRuntime_UClibc` is reserved and cannot be set
+fail_compilation/reserved_version.d(190): Error: version identifier `D_Coverage` is reserved and cannot be set
+fail_compilation/reserved_version.d(191): Error: version identifier `D_Ddoc` is reserved and cannot be set
+fail_compilation/reserved_version.d(192): Error: version identifier `D_InlineAsm_X86` is reserved and cannot be set
+fail_compilation/reserved_version.d(193): Error: version identifier `D_InlineAsm_X86_64` is reserved and cannot be set
+fail_compilation/reserved_version.d(194): Error: version identifier `D_LP64` is reserved and cannot be set
+fail_compilation/reserved_version.d(195): Error: version identifier `D_X32` is reserved and cannot be set
+fail_compilation/reserved_version.d(196): Error: version identifier `D_HardFloat` is reserved and cannot be set
+fail_compilation/reserved_version.d(197): Error: version identifier `D_SoftFloat` is reserved and cannot be set
+fail_compilation/reserved_version.d(198): Error: version identifier `D_PIC` is reserved and cannot be set
+fail_compilation/reserved_version.d(199): Error: version identifier `D_SIMD` is reserved and cannot be set
+fail_compilation/reserved_version.d(200): Error: version identifier `D_Version2` is reserved and cannot be set
+fail_compilation/reserved_version.d(201): Error: version identifier `D_NoBoundsChecks` is reserved and cannot be set
+fail_compilation/reserved_version.d(204): Error: version identifier `all` is reserved and cannot be set
+fail_compilation/reserved_version.d(205): Error: version identifier `none` is reserved and cannot be set
```

Comment:
That's a large diff for little value. Just put a `#line` directive down here.

Reply:
I would still need to change line-numbers after Emscripten and WASM32. Perhaps it would be better to add them at the end?

Reply:
Yep at the end sounds good